src/constants/errors.js

```diff
@@ -0,0 +1,4 @@
+export const NO_RESULT = 'NO_RESULT'
```

Comment:
Could we get a top-level comment about the intentions of this file? Is it intended for _any_ error in the application or errors specific to media fetching?

Reply:
This is for any error "fatal" that might arise i.e. any error that leads to the content being replaced by the image + explanation view.

Reply:
Added an explanatory comment to the file in 435c3bb.